flake.nix

```
@@ -3,7 +3,9 @@
 
   inputs = {
     utils.url = "github:numtide/flake-utils";
-    nixpkgs.url = "github:nixos/nixpkgs";
+    rv-utils.url = "github:runtimeverification/rv-nix-tools/f3e8bab938db9800044a94b94e59c5eba25977bc";
```

i think its fine to just do
```suggestion
    rv-utils.url = "github:runtimeverification/rv-nix-tools";
```
since we fully control updates to this repo and it will make the update scripts a lot easier (we wont have to sed the hash inside the nix file and can just call `nix flake lock --update-input rv-utils`